### PR TITLE
i mapped things: HD remaster

### PR DESCRIPTION
**Pull request by Shnupbups**
now with significantly less being bad and incompatible with #1269

### CODE REVIEW

File: mappings/net/minecraft/world/biome/BasaltDeltaBiome.mapping [hunk 1]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5152 net/minecraft/world/biome/BasaltDeltaBiome
```
Comment (YanisBft) [inline mappings/net/minecraft/world/biome/BasaltDeltaBiome.mapping]:
```suggestion
CLASS net/minecraft/class_5152 net/minecraft/world/biome/BasaltDeltasBiome
```

Comment (Shnupbups) [reply mappings/net/minecraft/world/biome/BasaltDeltaBiome.mapping]: Is the biome officially called 'Basalt Deltas'?

Comment (liach) [reply mappings/net/minecraft/world/biome/BasaltDeltaBiome.mapping]: yes, its official name includes a `s`. Don't apply this suggestion but rename in enigma (the file needs a rename as well)

Comment (YanisBft) [reply mappings/net/minecraft/world/biome/BasaltDeltaBiome.mapping]: Yep, it's the id

**Comment by Shnupbups** [comment]
thank you yanis, very cool

File: mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping [hunk 2]
```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/class_4790 net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder
-	FIELD field_22204 GRAVEL Lnet/minecraft/class_2680;
-	FIELD field_22209 SOUL_SAND Lnet/minecraft/class_2680;
+	FIELD field_22204 VALLEY_GRAVEL Lnet/minecraft/class_2680;
+	FIELD field_22209 VALLEY_SOUL_SAND Lnet/minecraft/class_2680;
```
Comment (YanisBft) [inline mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping]: Why?

File: mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping [hunk 3]
```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_5156 net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig
+	FIELD field_23887 minimumReach I
+	FIELD field_23888 maximumReach I
+	FIELD field_23889 minimumHeight I
+	FIELD field_23890 maximumHeight I
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping]:
```suggestion
	FIELD field_23887 minReach I
	FIELD field_23888 maxReach I
	FIELD field_23889 minHeight I
	FIELD field_23890 maxHeight I
```

File: mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	METHOD <init> (IIII)V
+		ARG 1 minimumReach
+		ARG 2 maximumReach
+		ARG 3 minimumHeight
+		ARG 4 maximumHeight
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping]:
```suggestion
		ARG 1 minReach
		ARG 2 maxReach
		ARG 3 minHeight
		ARG 4 maxHeight
```

File: mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+	METHOD method_27109 deserialize (Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_5156;
+	CLASS class_5157 Builder
+		FIELD field_23891 minimumReach I
+		FIELD field_23892 maximumReach I
+		FIELD field_23893 minimumHeight I
+		FIELD field_23894 maximumHeight I
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping]:
```suggestion
		FIELD field_23891 minReach I
		FIELD field_23892 maxReach I
		FIELD field_23893 minHeight I
		FIELD field_23894 maxHeight I
```

File: mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+		METHOD method_27110 build ()Lnet/minecraft/class_5156;
+		METHOD method_27111 reach (I)Lnet/minecraft/class_5156$class_5157;
+			ARG 1 reach
+		METHOD method_27112 reach (II)Lnet/minecraft/class_5156$class_5157;
+			ARG 1 minimum
+			ARG 2 maximum
+		METHOD method_27113 height (II)Lnet/minecraft/class_5156$class_5157;
+			ARG 1 minimum
+			ARG 2 maximum
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping]:
```suggestion
			ARG 1 min
			ARG 2 max
```

File: mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+		FIELD field_23891 minimumReach I
+		FIELD field_23892 maximumReach I
+		FIELD field_23893 minimumHeight I
+		FIELD field_23894 maximumHeight I
+		METHOD method_27110 build ()Lnet/minecraft/class_5156;
+		METHOD method_27111 reach (I)Lnet/minecraft/class_5156$class_5157;
+			ARG 1 reach
+		METHOD method_27112 reach (II)Lnet/minecraft/class_5156$class_5157;
+			ARG 1 minimum
+			ARG 2 maximum
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/BasaltColumnsFeatureConfig.mapping]:
```suggestion
			ARG 1 min
			ARG 2 max
```

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 8]
```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/class_5158 net/minecraft/world/gen/feature/DeltaFeatureConfig
+	FIELD field_23895 contents Lnet/minecraft/class_2680;
+	FIELD field_23896 rim Lnet/minecraft/class_2680;
+	FIELD field_23897 minimumRadius I
+	FIELD field_23898 maximumRadius I
+	FIELD field_23899 maximumRim I
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping]:
```suggestion
	FIELD field_23897 minRadius I
	FIELD field_23898 maxRadius I
	FIELD field_23899 maRim I
```

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+	METHOD <init> (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;III)V
+		ARG 1 contents
+		ARG 2 rim
+		ARG 3 minimumRadius
+		ARG 4 maximumRadius
+		ARG 5 maximumRim
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping]:
```suggestion
		ARG 3 minRadius
		ARG 4 maxRadius
		ARG 5 maxRim
```

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+	METHOD method_27114 deserialize (Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_5158;
+	CLASS class_5159 Builder
+		FIELD field_23900 contents Ljava/util/Optional;
+		FIELD field_23901 rim Ljava/util/Optional;
+		FIELD field_23902 minimumRadius I
+		FIELD field_23903 maximumRadius I
+		FIELD field_23904 maximumRim I
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping]:
```suggestion
		FIELD field_23902 minRadius I
		FIELD field_23903 maxRadius I
		FIELD field_23904 maxRim I
```

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+		METHOD method_27115 build ()Lnet/minecraft/class_5158;
+		METHOD method_27116 radius (II)Lnet/minecraft/class_5158$class_5159;
+			ARG 1 minimum
+			ARG 2 maximum
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping]:
```suggestion
			ARG 1 min
			ARG 2 max
```

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+		METHOD method_27117 contents (Lnet/minecraft/class_2680;)Lnet/minecraft/class_5158$class_5159;
+			ARG 1 contents
+		METHOD method_27118 rim (Lnet/minecraft/class_2680;I)Lnet/minecraft/class_5158$class_5159;
+			ARG 1 rim
+			ARG 2 maximum
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping]:
```suggestion
			ARG 2 maxRim
```

File: mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping [hunk 13]
```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_5160 net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig
+	FIELD field_23905 target Lnet/minecraft/class_2680;
+	FIELD field_23906 state Lnet/minecraft/class_2680;
+	FIELD field_23907 minimumReachPos Lnet/minecraft/class_2382;
+	FIELD field_23908 maximumReachPos Lnet/minecraft/class_2382;
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping]:
```suggestion
	FIELD field_23907 minReachPos Lnet/minecraft/class_2382;
	FIELD field_23908 maxReachPos Lnet/minecraft/class_2382;
```

File: mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+	METHOD <init> (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;Lnet/minecraft/class_2382;Lnet/minecraft/class_2382;)V
+		ARG 1 target
+		ARG 2 state
+		ARG 3 minimumReachPos
+		ARG 4 maximumReachPos
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping]:
```suggestion
		ARG 3 minReachPos
		ARG 4 maxReachPos
```

File: mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+	METHOD method_27119 deserialize (Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_5160;
+	CLASS class_5161 Builder
+		FIELD field_23909 target Lnet/minecraft/class_2680;
+		FIELD field_23910 state Lnet/minecraft/class_2680;
+		FIELD field_23911 minimumReachPos Lnet/minecraft/class_2382;
+		FIELD field_23912 maximumReachPos Lnet/minecraft/class_2382;
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping]:
```suggestion
		FIELD field_23911 minReachPos Lnet/minecraft/class_2382;
		FIELD field_23912 maxReachPos Lnet/minecraft/class_2382;
```

File: mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+		METHOD method_27120 build ()Lnet/minecraft/class_5160;
+		METHOD method_27121 target (Lnet/minecraft/class_2680;)Lnet/minecraft/class_5160$class_5161;
+			ARG 1 target
+		METHOD method_27122 minimumReachPos (Lnet/minecraft/class_2382;)Lnet/minecraft/class_5160$class_5161;
+			ARG 1 minimumReachPos
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping]:
```suggestion
		METHOD method_27122 minReachPos (Lnet/minecraft/class_2382;)Lnet/minecraft/class_5160$class_5161;
			ARG 1 minReachPos
```

File: mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+		METHOD method_27123 state (Lnet/minecraft/class_2680;)Lnet/minecraft/class_5160$class_5161;
+			ARG 1 state
+		METHOD method_27124 maximumReachPos (Lnet/minecraft/class_2382;)Lnet/minecraft/class_5160$class_5161;
+			ARG 1 maximumReachPos
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/NetherrackReplaceBlobsFeatureConfig.mapping]:
```suggestion
		METHOD method_27124 maxReachPos (Lnet/minecraft/class_2382;)Lnet/minecraft/class_5160$class_5161;
			ARG 1 maxReachPos
```

File: mappings/net/minecraft/world/gen/surfacebuilder/BasaltDeltasSurfaceBuilder.mapping [hunk 18]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5163 net/minecraft/world/gen/surfacebuilder/BasaltDeltasSurfaceBuilder
+	FIELD field_23915 DELTA_BASALT Lnet/minecraft/class_2680;
+	FIELD field_23916 DELTA_BLACKSTONE Lnet/minecraft/class_2680;
+	FIELD field_23917 DELTA_GRAVEL Lnet/minecraft/class_2680;
```
Comment (liach) [inline mappings/net/minecraft/world/gen/surfacebuilder/BasaltDeltasSurfaceBuilder.mapping]:
```suggestion
	FIELD field_23915 BASALT Lnet/minecraft/class_2680;
	FIELD field_23916 BLACKSTONE Lnet/minecraft/class_2680;
	FIELD field_23917 GRAVEL Lnet/minecraft/class_2680;
```

File: mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping (hunk 2 above)
Comment (liach) [inline mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping]:
```suggestion
	FIELD field_22204 GRAVEL Lnet/minecraft/class_2680;
	FIELD field_22209 SOUL_SAND Lnet/minecraft/class_2680;
```

Comment (Shnupbups) [reply mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping]: `GRAVEL` and `SOUL_SAND` already exist in a superclass as final, thus these need unique names.

Comment (Shnupbups) [reply mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping]: `GRAVEL` and `SOUL_SAND` already exist in a superclass as final, thus these need unique names.

File: mappings/net/minecraft/world/gen/surfacebuilder/BasaltDeltasSurfaceBuilder.mapping (hunk 18 above)
Comment (Shnupbups) [reply mappings/net/minecraft/world/gen/surfacebuilder/BasaltDeltasSurfaceBuilder.mapping]: `GRAVEL`, `BASALT` and `BLACKSTONE` already exist in a superclass as final, thus these need unique names.

File: mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping (hunk 2 above)
Comment (Juuxel) [reply mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping]: Superclass static final fields can be shadowed with the same name.

Comment (liach) [reply mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping]: hmm, java allows you to declare static final fields/methods with same names in classes, and if super class fields are private you can safely rename them manually out of enigma

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 19]
```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/class_5158 net/minecraft/world/gen/feature/DeltaFeatureConfig
+	FIELD field_23895 contents Lnet/minecraft/class_2680;
+	FIELD field_23896 rim Lnet/minecraft/class_2680;
+	FIELD field_23897 minimumRadius I
```
Comment (Shnupbups) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping:4]:
```suggestion
	FIELD field_23897 minRadius I
	FIELD field_23898 maxRadius I
	FIELD field_23899 maxRim I
```

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 20]
```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/class_5158 net/minecraft/world/gen/feature/DeltaFeatureConfig
+	FIELD field_23895 contents Lnet/minecraft/class_2680;
+	FIELD field_23896 rim Lnet/minecraft/class_2680;
+	FIELD field_23897 minRadius I
+	FIELD field_23898 maxRadius I
+	FIELD field_23899 maRim I
```
Comment (Shnupbups) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping]:
```suggestion
	FIELD field_23899 maxRim I
```

File: mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping (hunk 2 above)
Comment (Shnupbups) [inline mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping]:
```suggestion
	FIELD field_22204 GRAVEL Lnet/minecraft/class_2680;
	FIELD field_22209 SOUL_SAND Lnet/minecraft/class_2680;
```

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Should try to get this merged before another wednesday and conflicts

**Review by jaskarth — APPROVED** [review]

**Pull request merged**